vcenter/vcenter.py

```diff
@@ -646,7 +656,11 @@ def get_machine_info(self, uuid):
             return result
 
     def wait_for_task(self, task):
-        while (task.info.state == 'running' or task.info.state == 'queued'):
+        state = None
+        while (True):
+            state = task.info.state
+            if state == 'success' or state == 'error':
+                break
```

Comment:
I think it is necessary to add here comment that this function minimizes the interaction with API, resulting in "ugly" code, but more stability.

Otherwise I am sure I would refactor it after month, forgotting about the original purpose.

Reply:
ok